+nosnoc/+discrete_problem/stewart.m

```diff
@@ -0,0 +1,771 @@
+classdef stewart < vdx.problems.Mpcc
```

Comment:
It is clear from the fully qualified name `nosnoc.dcs.stewart` vs `nosnoc.discrete_time_problem.stewart` though I agree if using an editor that does not automatically suffix with the parent directory for files with the same name this can get annoying. I don't have a much better naming scheme however.